### ISSUE TITLE
MI Gateway ZNDMWG03LM not working

### ISSUE THREAD

**Issue by nimbleconsulting** [Maxmudjon/com.maxmudjon.mihomey#23]
If I add this gateway it's been registered as an IR device.

**Comment by Maxmudjon** [comment]
I will add this feature later, the firmware should be no higher than v1.4.6_0012

**Comment by nimbleconsulting** [comment]
Thanks, it’s 1.4.7. Can I downgrade that?

On 31 Oct 2020, at 21:50, Maxmudjon <notifications@github.com> wrote:

﻿

I will add this feature later, the firmware should be no higher than v1.4.6_0012

**Comment by Maxmudjon** [comment]
No

**Comment by PeetersJan** [comment]
Downgrade might be possible after all:
https://github.com/AlexxIT/XiaomiGateway3/issues/87

**Comment by nimbleconsulting** [comment]
Thanks, that looks really technical :S

Van: PeetersJan <notifications@github.com>
Verzonden: Wednesday, 11 November 2020 10:58
Aan: Maxmudjon/com.maxmudjon.mihomey <com.maxmudjon.mihomey@noreply.github.com>
CC: Nick van den Broek <Nick@nimble.consulting>; Author <author@noreply.github.com>
Onderwerp: Re: [Maxmudjon/com.maxmudjon.mihomey] MI Gateway ZNDMWG03LM not working (#23)


Downgrade might be possible after all:
AlexxIT/XiaomiGateway3#87<https://github.com/AlexxIT/XiaomiGateway3/issues/87>

**Comment by PeetersJan** [comment]
Indeed.  Condider this a first attempt. Probably earier methods will follow.

**Comment by nimbleconsulting** [comment]
I will add this feature later, the firmware should be no higher than v1.4.6_0012

What's the timeline on this?

**Comment by nimbleconsulting** [comment]
> I will add this feature later, the firmware should be no higher than v1.4.6_0012
> 
> What's the timeline on this?

Thanks! Do you have any idea when? I now have one with 1.4.5 :)
